content/en/community/hackathons/2023-03-athens/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Athens 2023
+date: 2023-01-20T07:20:00+02:00
+draft: false
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.png
+---
+
+## Athens Unikraft Hackathon
+
+Unikraft with [Nubificus](https://nubificus.co.uk/), [the Institute of Communication and Computer Systems (ICCS)](https://www.iccs.gr/en/) and [the National Technical University of Athens (NTUA)](http://www.ntua.gr/index_en.html) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.
+
+The hackathon will take place as an in-person event at the Multimedia Amphitheatre at the Zografos campus of NTUA.
+Address is TODO
+TODO detailed instructions on reaching the location of the hackathon
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.
+
```

Comment:
Please bring your own laptop. It’s best if you have a native Linux install on your laptop. For any situation, we will have virtual machines ready, to be able to get hacking quickly and easily!